content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,165 @@
+# Build and Run `rippled` on Ubuntu
+
+`rippled` is the core peer-to-peer server that manages the XRP Ledger. A `rippled` server can connect to a network of peers, relay cryptographically signed transactions, and maintain a local copy of the complete shared global ledger.
+
+For an overview of `rippled`, see [Operating rippled Servers](tutorial-rippled-setup.html).
+
+Use these instructions to build a `rippled` binary file and run it as a stock `rippled` server on Ubuntu 15.04 or later. These instructions were tested on Ubuntu 16.04 LTS.
+
+For information about building `rippled` for other platforms, see [Builds](https://github.com/ripple/rippled/tree/develop/Builds) in the `rippled` GitHub repository.
+
+
+## System Requirements
+
+**_To build `rippled`:_**
+
+You need a **minimum** of 8GB of RAM.
+
+**_To run `rippled`:_**
+
+Meet these [system requirements](tutorial-rippled-setup.html#system-requirements).
+
+
+## 1. Build `rippled`
+
+These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the software prerequisites you need to build and run `rippled`.
+
+1. Update the list of packages that are available for `apt-get` to install or upgrade.
+
+        sudo apt-get update
+
+2. Upgrade currently installed packages.
+
+        sudo apt-get -y upgrade
+
+3. Install Git.
+
+        sudo apt-get -y install git
+
+4. Install SCons.
+
+        sudo apt-get -y install scons
+
+5. Install Ctags.
+
+        sudo apt-get -y install ctags
```

Comment:
Why ctags?

Reply:
Thank you for calling this out. Indeed, installation of ctags is optional. It is a tool that a developer, in general, may want to install, but ctags is not specifically required by rippled.  Confirmed that build and run works fine without it. Removing this step.